### ISSUE TITLE
 InvalidArgumentError 

### ISSUE THREAD

**Issue by Leeking7012** [JinyuZ1996/TiDA-GCN#1]
Hi,i met this problem when running this TiDA-GCN code

InvalidArgumentError (see above for traceback): indices[120,6] = 19771 is not in [0, 11405)
      
I searched for a lot and i know there's an error in embedding layer
can you help me solve this problem?

**Comment by JinyuZ1996** [comment]
We're very sorry for not paying attention to your message promptly. If you still cannot solve the above problem, please provide a detailed description of the specific situation when you encountered the problem, and we will do our best to help you address it.

**Comment by JinyuZ1996** [comment]
Please consider answering the following questions so that we can help you solve the issue:1) have you configured the class library of the code according to the version provided by us? 2) Are you experiencing problems while reproducing our code or trying to modify it? 3）Did you use the dataset provided by us or your own dataset for experiments?

**Comment by muyunping123** [comment]
hello，i met the same error like with the above description.

> Hi,i met this problem when running this TiDA-GCN code
> 
> InvalidArgumentError (see above for traceback): indices[120,6] = 19771 is not in [0, 11405)
> 
> I searched for a lot and i know there's an error in embedding layer can you help me solve this problem?

From my code, i used the same source code and the datasets. 
The error is as follows:
`InvalidArgumentError (see above for traceback): indices[120,6] = 19771 is not in [0, 11405)
	 [[node encoder/SA_B/embedding_lookup (defined at D:\Code\Pycharm\Sharedaccount\compare\compare_tidagcn.py:953)  = GatherV2[Taxis=DT_INT32, Tindices=DT_INT32, Tparams=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:CPU:0"](encoder/concat_3, _arg_inputs/seq_B_0_6, encoder/SA_B/embedding_lookup/axis)]]`
The code where occurring this error is the line `        with tf.variable_scope('SA_B', reuse=tf.AUTO_REUSE):

            masks_B = tf.expand_dims(tf.to_float(tf.not_equal(seq_B, self.n_items_B)), -1)
            **seq_emb_B = tf.nn.embedding_lookup(i_embeddings_B, seq_B)**
            seq_emb_B_output_1 = seq_emb_B` 
Can you help me to fix it, thanks!
